Xamarin.Forms.Core/Page.cs

```diff
@@ -57,6 +57,7 @@ public class Page : VisualElement, ILayout, IPageController, IElementConfigurati
 
 		public Page()
 		{
+			this.Dispatcher = DispatcherManager.Current.Dispatcher;
```

Comment:
@wachs Apologies if I've asked this before - since Page inherits from BindableObject, is this line redundant?

Reply:
It really ends up not being necessary.